apps/omg/lib/omg/root_chain_coordinator.ex

```diff
@@ -106,7 +106,7 @@ defmodule OMG.RootChainCoordinator do
   end
 
   def handle_call({:check_in, synced_height, service_name}, {pid, _ref}, state) do
-    _ = Logger.debug("#{inspect(service_name)} checks in on height #{inspect(synced_height)}")
+    _ = Logger.info("#{inspect(service_name)} checks in on height #{inspect(synced_height)}")
```

Comment:
I don't think you want to do this, this creates a lot of noise.

Reply:
then we delete those you think we don't need. I am actually doing this PR blindly 😛

Reply:
if you want to remove logs from production code you can do purging of a specific level, https://hexdocs.pm/logger/Logger.html#module-application-configuration
compile_time_purge_matching 

I would suggest you enable all these logs locally, build your cluster with dockercompose and run cabbage tests :)